modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/OnboardingLoginOrSignUpPage.kt

```diff
@@ -99,29 +108,67 @@ internal fun OnboardingLoginOrSignUpPage(
 
             Spacer(Modifier.height(8.dp))
 
-            OutlinedRowButton(
-                text = stringResource(LR.string.onboarding_continue_with_google),
-                leadingIcon = IR.drawable.google_g,
-                onClick = onContinueWithGoogleClicked
-            )
-
-            RowButton(
-                text = stringResource(LR.string.onboarding_sign_up_free),
-                onClick = onSignUpFreeClicked
-            )
-
-            OutlinedRowButton(
-                text = stringResource(LR.string.log_in),
-                onClick = onLoginClicked
-            )
+            ContinueWithGoogleButton(onClick = onContinueWithGoogleClicked)
+            SignUpButton(onClick = onSignUpFreeClicked)
+            LogInButton(onClick = onLoginClicked)
         }
     }
 }
 
-@Preview
+private val continueWithGoogleBorder: BorderStroke
+    @Composable
+    get() = BorderStroke(2.dp, MaterialTheme.theme.colors.primaryInteractive03)
+
+private val continueWithGoogleColors: ButtonColors
+    @Composable
+    get() = ButtonDefaults.outlinedButtonColors(contentColor = MaterialTheme.theme.colors.primaryText01)
```

Comment:
Are you meant to put these are file variables? I wasn't sure if it was costly in Compose or not.

Reply:
I don't think it would cause a performance issue. I would include this as a variable declaration in the `SignUpButton` composable function myself since that's the only place it's used, but I don't feel strongly.

Reply:
Thanks, I'm going to follow what you do and have put this in the method now.